### PR TITLE
Feature/guild dashboard

### LINKED ISSUE #110: Guild dashboard
Provide a dashboard similar to the developer stats dashboard

This dashboard should contain:
Top 3 Reputation users
Total guild reputation
Week guild reputation
Today guild reputation
Top reputation channel

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Introduces a guild dashboard which provides basic insight into guild statistics.

Closes #110

### CODE REVIEW

File: src/main/resources/locale_fr_FR.properties [hunk 1]
```diff
@@ -12,6 +12,13 @@ command.channel.sub.set=Définir le canal de réputation
 command.channel.sub.set.set=Définit le canal suivant comme canal de réputation :\n%CHANNEL%
 command.channel.sub.whitelist.false=Aucune Réputation ne peut être collectée dans les canaux répertoriés.
 command.channel.sub.whitelist.true=La réputation ne peut être collectée que dans les canaux répertoriés.
+command.dashboard.description=Obtenir des informations sur ce serveur.
+command.dashboard.title=Informations sur le %GUILD
```
Comment (Taucher2003) [inline src/main/resources/locale_fr_FR.properties:16]:
```suggestion
command.dashboard.title=Informations sur le %GUILD%
```

**Review by Taucher2003 — CHANGES_REQUESTED** [review]

**Review by Taucher2003 — APPROVED** [review]

File: src/main/resources/locale.properties [hunk 2]
```diff
@@ -12,6 +12,13 @@ command.channel.sub.set=Set the reputation channel
 command.channel.sub.set.set=Set the following channel as reputation channel:\n%CHANNEL%
 command.channel.sub.whitelist.false=No Reputation can be collected in listed channels.
 command.channel.sub.whitelist.true=Reputation can only be collected in listed channels.
+command.dashboard.description=Get information about this server.
+command.dashboard.title=%GUILD% Information
+command.dashboard.todayReputation=Today Reputation
```
Comment (dlsf) [inline src/main/resources/locale.properties:17]:
```suggestion
command.dashboard.todayReputation=Reputation today
```

Comment (dlsf) [reply src/main/resources/locale.properties:17]:
```suggestion
command.dashboard.todayReputation=Todays Reputation
```

Comment (dlsf) [reply src/main/resources/locale.properties:17]: Choose your fighter

File: src/main/resources/locale_de_DE.properties [hunk 3]
```diff
@@ -12,6 +12,13 @@ command.channel.sub.set=Setzen des Reputationskanals
 command.channel.sub.set.set=Setzt den folgenden Kanal als Reputationskanal:\n%CHANNEL%
 command.channel.sub.whitelist.false=In gelisteten Kanälen kann keine Reputation gesammelt werden.
 command.channel.sub.whitelist.true=Nur in gelisteten Kanälen kann Reputation gesammelt werden.
+command.dashboard.description=Erhalten Informationen über diesen Server.
```
Comment (dlsf) [inline src/main/resources/locale_de_DE.properties:15]: Other command descriptions don't directly address the reader. Additionally, this is grammatically wrong.
```suggestion
command.dashboard.description=Zeigt Informationen über diesen Server an.
```

File: src/main/resources/locale_en_US.properties [hunk 4]
```diff
@@ -12,6 +12,13 @@ command.channel.sub.set=Set the reputation channel
 command.channel.sub.set.set=Set the following channel as reputation channel:\n%CHANNEL%
 command.channel.sub.whitelist.false=No Reputation can be collected in listed channels.
 command.channel.sub.whitelist.true=Reputation can only be collected in listed channels.
+command.dashboard.description=Get information about this server.
+command.dashboard.title=%GUILD% Information
+command.dashboard.todayReputation=Today Reputation
```
Comment (dlsf) [inline src/main/resources/locale_en_US.properties:17]:
```suggestion
command.dashboard.todayReputation=Todays Reputation
```

Comment (dlsf) [reply src/main/resources/locale_en_US.properties:17]:
```suggestion
command.dashboard.todayReputation=Reputation today
```

**Review by dlsf — CHANGES_REQUESTED** [review]

**Review by dlsf — APPROVED** [review]

**Pull request merged**